packages/node/src/integrations/http.ts

```diff
@@ -90,7 +103,26 @@ type WrappedRequestMethodFactory = (original: OriginalRequestMethod) => WrappedR
 function _createWrappedRequestMethodFactory(
   breadcrumbsEnabled: boolean,
   tracingEnabled: boolean,
+  tracePropagationTargets: (string | RegExp)[] | undefined,
```

Comment:
optional/question: Does it make sense to extract `(string | RegExp)[] | undefined` to a type?

Reply:
Yeah that makes a lot of sense! Can you check whether the location where I extracted it makes sense?: 0aa153f929321dbc0a591c9a3c5c67ee0b4f76d5

Also I extracted it without `| undefined` because that works better with the optional operator in the init options.

Reply:
I think it sort of makes sense but I can see why this might look a little off. Seems like we don't really have a tracing-related file in the types package. Adding it to the transaction, span or baggage files also doesn't make more sense... 

So wdyt about adding a new `tracing.ts` file? Or we add it to the `misc.ts` file ?

Reply:
Created a tracing.ts file. Thanks for the input!